lib/git_pair/cli.ex

```diff
@@ -17,10 +17,12 @@ defmodule GitPair.CLI do
   ]
 
   def main(argv) do
-    argv
+    {_result, message} = argv
     |> parse_args
     |> parse_command
     |> execute_command
+
+    IO.puts(message)
```

Comment:
Shouldn't we sanitize the message, at least for errors, before print?

Reply:
I think that initially we can start just printing it as it is, then we can create a new issue for sanitizing it.